include/create.hpp

```diff
@@ -28,6 +29,17 @@ int gotoLabOrTheory;
 bool goHome = false;
 
 // Function
+bool IsNumber(std::string s)            /* Function for checking string values if it has integer value or not */
+      {
+          for (int i = 0; i < s.length(); i++)
+              if (isdigit(s[i]) == false)
+                  return false;
+      
+          return true;
+      }
+
+
+// The main driver Func
```

Comment:
this is main function of this create.hpp file. This function creates folders and take all the necessary inputs